### PR TITLE
Small fix for the keyword validation

### PR DESCRIPTION
**Pull request by verveerpj**
Fixes a bug where validation of keywords that do not have a default value fails.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Review by joakim-hove — APPROVED** [review]

**Comment by verveerpj** [comment]
jenkins build this please

File: opm/simulators/flow/KeywordValidation.cpp [hunk 1]
```diff
@@ -129,14 +129,16 @@ namespace KeywordValidation
...
                     if (item_properties != keyword_properties->second.end()) {
-                        // Validate the item, if it is partially supported.
-                        validateKeywordItem<T>(keyword,
-                                               item_properties->second,
-                                               keyword.size() > 1,
-                                               record_index,
-                                               item_index,
-                                               item.get<T>(0),
-                                               errors);
+                        if (item.hasValue(0) && !item.defaultApplied(0)) {
```
Comment (joakim-hove) [inline opm/simulators/flow/KeywordValidation.cpp:132]: I see no reason no to add the `!item.defaultApplied(0)` here - it is not totally insane with a situation where the default is actually not supported.

Comment (verveerpj) [reply opm/simulators/flow/KeywordValidation.cpp]: Done.

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**